components/automate-chef-io/content/docs/desktop.md

```diff
@@ -13,7 +13,7 @@ Desktop information populates this dashboard after a Chef Infra Client run has e
 ## Setting Up the Desktop Dashboard
 
 Enable the Desktop dashboard with: `chef-automate deploy --product automate --product infra-server --product desktop`.
-For more information, see [Install Chef Infra Server with Automate](https://automate.chef.io/docs/infra-server/).
+For more information, see [Install Chef Infra Server with Chef Automate](https://automate.chef.io/docs/infra-server/).
```

Comment:
More information about what?

Reply:
installation information. Will file an issue to address further.